valhalla/jawn/src/lib/stores/experimentStore.ts

```diff
@@ -215,6 +260,12 @@ async function enrichExperiment(
     }
   }
 
+  const experimentScores = getExperimentScores(experiment);
+
+  if (!experimentScores.error && experimentScores.data) {
```

Comment:
This is kind of a nit, but i think this something as you progress your career is to feel very allergic to mutations. These things can cause many bugs. as more code get's inected. 

I would suggest a different pattern...

```ts
if (!experimentScores.error && experimentScores.data) {
   return {
      scores: experimentScores.data,
      ...experiment
   }
} else {
   return experiment;
}
```


This is not a big deal, but just something I wanna bring up now for future reference

Reply:
fixed